builtin/logical/aws/path_static_roles.go

```diff
@@ -108,7 +111,7 @@ func (b *backend) pathStaticRolesRead(ctx context.Context, req *logical.Request,
 
 	var config staticRoleConfig
 	if err := entry.DecodeJSON(&config); err != nil {
-		return nil, fmt.Errorf("failed to decode configuration for tatic role '%q': %w", roleName, err)
+		return nil, fmt.Errorf("failed to decode configuration for static role '%q': %w", roleName, err)
```

Comment:
I noticed you used `%q` here but wrapped it in single quotes, so it'd have 3 quotation marks around it. I'm assuming you simply want the `%q` without the additional quotes?